### ISSUE TITLE
Accessibility (a11y) issues for 5calls

### ISSUE THREAD

**Issue by kimflournoy** [5calls/5calls#56]
Here's what I noticed so far (and will work on fixing), in no particular order:

- No language set for the document
- Issues list: list items are not accessible at all (href doesn’t belong on a list item tag) - _critical_
- Done image (done.png) doesn’t have alt text
- representative images should have (blank) alt text
- add hidden text to main “call outcomes” buttons for better clarity
- “Change location” link needs more hidden text for clarity. Maybe it should also be a button?
- Aria live region should be set on main content area, call__promote # of calls left area, anywhere there's dynamic text I think
- Call Reason should not all be inside h3 tag
- screen reader reads h1 as “5 calls 5 calls” due to alt text on image. Best to have a blank alt attribute I think.

There are also some color contrast issues, as noted in https://github.com/5calls/5calls/issues/19

This is just from testing on Chrome with keyboard navigation and ChromeVox so far. I'll look at mobile later, once these are cleaned up.
